### PR TITLE
Section 3 Writeup

### PR DESCRIPTION
**Pull request by algilliland**
Most of outline in place, waiting to have Aropa access to add details regarding usage

### CODE REVIEW

**Comment by mstrout** [comment]
Can you take out the Section 2 writeup from the pull request?

File: Section3-PeerReview-Writeup/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Section3-PeerReview-WriteUp 
+
+## Introduction 
+The goal of today's section is to learn about peer review in regards to our class
+and code review in industry. In this section, you will be introduced to Aropa, our 
+peer review tool, and write your first two peer reviews.
```
Comment (mstrout) [inline Section3-PeerReview-Writeup/README.md]: the peer reviewing tool we will be using

File: Section3-PeerReview-Writeup/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+https://aropa2.gla.ac.uk/aropa/aropa.php
+
+You will be prompted to change your password. Choose a secure password that 
+you will remember as you will use this same account for peer reviews in 
+the future. 
+
+## The Assignment
+Overview : Upload your PA2 code for review and write two peer reviews for PA2. 
+
+### Upload Process
```
Comment (mstrout) [inline Section3-PeerReview-Writeup/README.md:41]: We are going to have to ask them to do this ahead of time.  Let's discuss Thursday night.

File: Section3-PeerReview-Writeup/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+### Review Process 
+You will be responding to the statements below in regards to two other
+students' code. You will be ranking your agreement of how well the statements 
+relate to the student's code on a scale of Strongly Agree to Strongly Disagree.
+
+In addition to the rankings, you will have a comment block below to add a
+rational for your ranking and commentary of how the student could improve. 
+When writing your comments, keep the Department of Computer Science's code of 
+conduct (https://www2.cs.arizona.edu/codeofconduct.pdf) in mind. Your feedback 
+should be constructive, polite and contribute to a positive learning environment.
```
Comment (mstrout) [inline Section3-PeerReview-Writeup/README.md:59]: Nice.  But put in the oxford comma after polite.

File: Section3-PeerReview-Writeup/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+3.	Decomposition: The student avoids nested loops or conditionals and chaining to 
+	self-defined methods. 
+	
+4.	Decomposition: The student's code exhibits good decomposition. They used all non-
+	static methods and only turned in one source file.
+
+5.	Style: The code follows the style guidelines. No lines are longer than 80 columns, does 
+	not mix tabs and spaces and has curly brackets on the same line as the loop or conditional. 
+
```
Comment (mstrout) [inline Section3-PeerReview-Writeup/README.md:76]: Then have an other suggestions field.

**Review by mstrout — CHANGES_REQUESTED** [review]
Please make these changes and update your fork with the main repository to get Section 2 writeup changes.

File: Section3-PeerReview-Writeup/README.md (hunk 4 above)
Comment (algilliland) [reply Section3-PeerReview-Writeup/README.md:76]: I am not certain of what you mean by this comment. 

If this is in regards to the comments boxes, I thought a comment box was to be placed after each of the 5 statements for the likert scale.

**Comment by mstrout** [comment]
Ali, this pull request includes some files from the main CS2018 repository.  I am a little concerned there is some local repository pollution going on.  Maybe you should email me the specific files you changed and I can update the repository?

**Pull request closed without merging**